src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -1681,7 +1681,7 @@ def download(cls, pretrained_model_name, **kwargs) -> Union[str, os.PathLike]:
                 if candidate_file in filenames:
                     custom_components[component] = module_candidate
                 elif module_candidate not in LOADABLE_CLASSES and not hasattr(pipelines, module_candidate):
-                    raise ValueError(
+                    logger.warn(
                         f"{candidate_file} as defined in `model_index.json` does not exist in {pretrained_model_name} and is not a module in 'diffusers/pipelines'."
                     )
 
```

Comment:
I don't think we should do this as this way, we cannot standardize the community component loading system. Since it's just about changing a single line in the `model_index.json` file, I believe it's okay this way. 

However, in https://github.com/huggingface/diffusers/pull/5556, I proposed to make the pipeline serialization process (that includes community components) even easier. 

@patrickvonplaten WDYT?

Reply:
@sayakpaul It seems that in my case, I also need to change the warehouse code structure? I changed _class_name to
```
"_class_name": [
    "stable_diffusion_reference_only",
    "pipelines",
    "StableDiffusionReferenceOnlyPipeline"
  ]
```
But prompt ```Could not locate the pipeline.py inside stable_diffusion_reference_only.```

Reply:
@sayakpaul It would be more elegant if you could add a parameter, such as specifying the class path of the pipeline in model.index, specifying the class path of the custom module in the custom module, and changing save_pretrained to automatically save.